### ISSUE TITLE
Long filename give os error 2 in windows 10

### ISSUE THREAD

**Issue by maskedfighter** [R3S3t9999/DoVi_Scripts#17]
Hi!
I have quite long filenames for my blurays, since the name contain a lot of information for grabbers, e.g. "A Quiet Place (2018) [imdbid-tt6644200] - [Remux-2160p][DV HDR10][AC3 5.1][h265].mkv". 
Whenever i want to 3-1 that it gives me the above mentioned or error 2 after some time of processing. This is really annoying since i have to rename every file before cm_analyze and then rename it back to it´s old filename.

**Comment by R3S3t9999** [comment]
yeah, some workflow will not work with special characters. It's usually a common issue with batch scripts and I did my best to make it work in most of the workflow. I'll look at your specific case but no promise.
